subprojects/docs/src/docs/release/notes.md

```diff
@@ -181,6 +181,10 @@ In order to see which toolchains got detected and their corresponding metadata,
 
 Output of `gradle -q javaToolchains`:
 ```
+ + Options
+     | Auto-detection enabled?  Yes
```

Comment:
Nice!
Alternative:
| Auto-detection:  Enabled

Reply:
👍 on this proposal

Reply:
Sounds good!